docs/react/week-3/lesson.md

```diff
@@ -444,81 +556,49 @@ function CreateAccountForm() {
 }
 ```
 
-We now have three different inputs named `username`, `email` and `password`, and we keep each entered value in state as a field in an object. The method `handleChange` is reused to keep track of changes for **all** the values. The trick here is to use the `name` of the `<input>` element to update the corresponding state.
+We now have three different inputs named `username`, `email` and `password`, and we keep each entered value in state as a field in an object.
```

Comment:
I don't see an object? It looks like we're using three state hooks in a function? Maybe reword?